compiler/rustc_ast/src/attr/mod.rs

```diff
@@ -360,7 +367,8 @@ impl MetaItem {
             _ => path.span.hi(),
         };
         let span = path.span.with_hi(hi);
-        Some(MetaItem { path, kind, span })
+        // FIX THIS LATER
```

Comment:
Can you expand this comment to give more information on what needs to be fixed?

Reply:
I've put a proper `FIXME` here. I expect I'll tackle this in a subsequent PR, so I'd prefer not having to block on this.

Reply:
Thanks, yes that works me.